site/content/events/2016-denver/_sponsors.txt

```diff
@@ -132,6 +139,28 @@ filter: erb
     :width  => '150px',
     :height => '50px'
 },
+{
+    :name   => 'SumoLogic',
+    :image  => 'sumologic.png',
+    :link   => 'https://www.sumologic.com/',
+    :width  => '150px',
+    :height => '50px'
+},
+{
```

Comment:
Doh. Good catch. I will fix it and push.

On Thu, Mar 3, 2016, 7:10 PM Michael Lanyon <notifications@github.com>
wrote:

> In site/content/events/2016-denver/_sponsors.txt
> <https://github.com/jedi4ever/devopsdays-webby/pull/2519#discussion_r54981810>
> :
>
> > @@ -132,6 +139,28 @@ filter: erb
> >      :width  => '150px',
> >      :height => '50px'
> >  },
> > +{
> > +    :name   => 'SumoLogic',
> > +    :image  => 'sumologic.png',
> > +    :link   => 'https://www.sumologic.com/',
> > +    :width  => '150px',
> > +    :height => '50px'
> > +},
> > +{
>
> two SumoLogics?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jedi4ever/devopsdays-webby/pull/2519/files#r54981810>.
>